rst_files/generic_programming.rst

```diff
@@ -564,7 +575,7 @@ Reals and Algebraic Structures
 Thinking back to the mathematical motivation, a `Field <https://en.wikipedia.org/wiki/Field_\(mathematics\)>`_ is an `Ring` with a few additional properties, among them
 
     * a multiplicative inverse: :math:`a^{-1}`
-    * an inverse operation for multiplication: :math:`a / b = a \cdot b^{-1}`
+    * an inverse operation for multiplication: :math:`\frac{a}{b} = a \cdot b^{-1}`
```

Comment:
This is only true for commutative fields, so I think we should change it back.

Reply:
I thought fields were always commutative... i.e., the elements are an abelian group under addition, and the units/invertibles are an abelian group under multiplication

Reply:
http://mathworld.wolfram.com/FieldAxioms.html

Reply:
Sorry, my brain was mixed up with the semi-group below.

Never theless, the ambigurity of the fraction means that I think `a / b` is preferrable in this instance, even if `b \ a = frac{a}{b}` would be equivalent

Reply:
All good, will revert the change.